Client/ui/modules/apps/SumoTimer/app.js

```diff
@@ -57,6 +57,41 @@ angular.module('beamng.apps')
                 updateTime();
             }
 
+            function animateCircleSize() {
+                const originalWidth = canvas.width;
+                const originalHeight = canvas.height;
+                const targetWidth = originalWidth * 1.1;
+                const targetHeight = originalHeight * 1.1;
+                const duration = 500; // Total duration in milliseconds for the animation
+                const stepTime = 20; // Time in milliseconds between each step
+                let currentStep = 0;
+                const totalSteps = duration / stepTime;
+
+                const sizeInterval = setInterval(() => {
+                    currentStep++;
+                    const stepRatio = currentStep / totalSteps;
+                    if (currentStep <= totalSteps / 2) {
+                        // Increasing size
+                        canvas.width = originalWidth + (targetWidth - originalWidth) * stepRatio * 2;
+                        canvas.height = originalHeight + (targetHeight - originalHeight) * stepRatio * 2;
+                    } else {
+                        // Decreasing size
+                        canvas.width = targetWidth - (targetWidth - originalWidth) * (stepRatio - 0.5) * 2;
+                        canvas.height = targetHeight - (targetHeight - originalHeight) * (stepRatio - 0.5) * 2;
+                    }
+
+                    // Redraw the circle and sector after resizing
+                    updateSector();
+
+                    if (currentStep >= totalSteps) {
+                        clearInterval(sizeInterval);
+                        canvas.width = originalWidth; // Reset to original size to avoid floating point errors
+                        canvas.height = originalHeight;
+                        updateSector(); // Final redraw to ensure proper display
+                    }
```

Comment:
```suggestion
                    if (currentStep >= totalSteps) {
                        clearInterval(sizeInterval);
                        canvas.width = originalWidth; // Reset to original size to avoid floating point errors
                        canvas.height = originalHeight;
                    }
                    updateSector();
```